02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,15 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Andrew Krummenacher</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <h1>Andrew Krummenacher</h1>
+    <img src="https://scontent.xx.fbcdn.net/v/t1.0-9/1915339_10153326470976993_5076233424777823214_n.jpg?oh=e02ff3e47d4243a955a19966d09fdc76&oe=581FD2FB" />
+  <h2>Bio</h2>
+  <h2>Interests</h2>
+  <h2>Hobbies</h2>
 
 </body>
 </html>
```

Comment:
This is a good start to your bio page Andrew. I would have liked to see you use more of the HTML elements we talked about in class.